package/boot/uboot-rockchip/patches/001-rockchip-Avoid-using-libfdt-with-of-platdata.diff

```diff
@@ -0,0 +1,65 @@
+diff --git a/drivers/clk/rockchip/clk_rk3328.c b/drivers/clk/rockchip/clk_rk3328.c
```

Comment:
I can only assume what this patch does. Is this required to build U-Boot without `python-dev`?